Webex Connect Flows/README.md

```diff
@@ -46,13 +48,5 @@ In 3.2 version of workflows, enhancements to the flows allow for setting contact
 #### Event Handling Flows:-
 - This folder contains event handling template flows. It contains Task Close Flow, Task Modified Flow, Task Routed Flow.
 
-### Steps to perform Manual Upgrade from v3.1 to v3.2
-- As described in file `Webex Connect Flows/How to manually upgrade from v3.1 to v3.2 workflows.MD`
-
-## Version History
-
-| Workflows version | High level Description                                                                                                                                                                                                                                  |
-|-------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| 3.2               | Flow nodes have been updated to enable setting contact priority.                                                                                                                                                                                        |
-| 3.1               | The malware scanning feature has been integrated into all attachments sent across various channels. The channel start nodes have been improved to include a new set of variables for identifying malware and presenting security scan results.          |
-| 3.0               | The 2.x flows undergone a major update in 3.0 where the contact center specific logic that used to be a part of flow got replaced with the Resolve Conversation node. Shared flows like Task Routed, Task Modified and Task Closed are also simplified. |
\ No newline at end of file
+### Steps to perform Manual Upgrade from v3.2 to v3.3
+- As described in file `Webex Connect Flows/How to manually upgrade from v3.2 to v3.3 workflows.MD`
```

Comment:
Please replace this with an internal link to the file

Reply:
Sure